### ISSUE TITLE
Typo in Lua interface example

### ISSUE THREAD

**Issue by jgoldfar** [shogun-toolbox/shogun#4038]
The Lua example here https://github.com/shogun-toolbox/shogun/blob/develop/doc/readme/INTERFACES.md appears to have a typo; the call to run the example should probably be closer to what is generated by https://github.com/shogun-toolbox/shogun/blob/develop/examples/meta/lua/CMakeLists.txt (I don't run lua locally, otherwise would submit a PR)

**Comment by karlnapf** [comment]
Whoops, indeed!
Thanks for reporting.

This is a lovely entrance task for those try to get involved in Shogun.

**Comment by karlnapf** [comment]
The R command also is not rendered properly, so that needs a mini fix as well

**Comment by prashanthduvvada** [comment]
I want to take up and work on this issue.

**Comment by karlnapf** [comment]
do it :)

**Comment by prashanthduvvada** [comment]
Well, do I need to create a separate issue and create different PR for R command?

**Comment by karlnapf** [comment]
Nope, just fixing typos can be done in a single PR which consists of 1-2 lines
